tests/integration/views/surveys/help/test_surveys_help_something_else.py

```diff
@@ -167,7 +167,6 @@ def test_create_message_post_something_else_failure(self, mock_request, get_surv
 
         self.assertEqual(response.status_code, 200)
         self.assertIn("Error: ".encode(), response.data)
-        self.assertIn('<span class="ons-panel__assistive-text ons-u-vh">Error: </span>'.encode(), response.data)
```

Comment:
Kept in some, but removed for this one?

Reply:
I've put this one back in, missed it while I was adding them back and fixing them